test/jdk/java/foreign/StdLibTest.java

```diff
@@ -354,7 +353,7 @@ private MethodHandle specializedPrintf(List<PrintfArg> args) {
                 argLayouts.add(arg.layout);
             }
             MethodHandle mh = abi.downcallHandle(printfAddr, mt,
-                    FunctionDescriptor.of(C_INT, false, argLayouts.toArray(new MemoryLayout[0])));
+                    FunctionDescriptor.of(C_INT, argLayouts.toArray(new MemoryLayout[0])));
```

Comment:
It feels like we should be able to simplify this routine with the new FunctionDescriptor::append ?

Reply:
Good catch! - updated